src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -169,6 +192,58 @@ def pex_requirements_for_default_lockfile(cls):
             resolve_name=cls.options_scope,
         )
 
+    @classproperty
+    def default_package_name_and_version(cls) -> Optional[PackageNameAndVersion]:
```

Comment:
Given one caller of this requires the result is not-None, maybe that assumption/assertion could be moved in here, so that this can return:

```suggestion
    def default_package_name_and_version(cls) -> PackageNameAndVersion:
```

And simplify the call sites?

Reply:
Done! I had left it optional before because some tests broke, but I think that's because they had no `help` field rather than having issues with the default lockfile.